### PR TITLE
docs: k3s add multipass macos install 

### PR DESCRIPTION
**Pull request by babadofar**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Checklist:

- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [ ] Any dependent changes have already been merged.

### CODE REVIEW

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 1]
```diff
@@ -29,14 +27,42 @@ Make sure you have created a cluster using k3s.
...
+# export kubeconfig
+export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
 # Set it also in the bashrc or zshrc file, or you can flatten both of these configs into a single file
 KUBECONFIG=~/.kube/config:~/.kube/k3s-config
+
+```
+
+​
+
+#### Mac Os
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:47]: Probably use `macOS`  https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/ or OS X (https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform)

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 2]
```diff
@@ -29,14 +27,42 @@ Make sure you have created a cluster using k3s.
...
+multipass shell k3sVMcurl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=v1.21 sh -s - --write-kubeconfig-mode 644
+
+# copy kubeconfig
+sudo mkdir -p ~/.kube/ &&  sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/k3s-config
+# export kubeconfig
+export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
+
 ```
 
+
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:65]: remove extra line.

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 3]
```diff
@@ -29,14 +27,42 @@ Make sure you have created a cluster using k3s.
...
+# install k3 with kubernetes version 1.21 (We don't support Kubernetes 1.22+ yet)
 curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=v1.21 sh -s - --write-kubeconfig-mode 644
+
+# copy kubeconfig
 sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/k3s-config
+# export kubeconfig
+export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
 # Set it also in the bashrc or zshrc file, or you can flatten both of these configs into a single file
 KUBECONFIG=~/.kube/config:~/.kube/k3s-config
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:41]: This line can be removed, as we are exporting KUBECONFIG before.

**Comment by babadofar** [comment]
This is now added to my other too-long-running PR https://github.com/jenkins-x/jx-docs/pull/3588

**Pull request closed without merging**